### ISSUE TITLE
active expression sometimes doesn't stay highlighted

### ISSUE THREAD

**Issue by epixode** [France-ioi/codecast#69]
While stepping through code, the highlighting of the active expression sometimes appears briefly before disappearing.

**Comment by remisharrock** [comment]
yes, here are details of this bug in french : 

[bug ligne verte 1](https://codecast.france-ioi.org/next/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Ftcom%2F1512646351964)

[bug ligne verte 2](https://codecast.france-ioi.org/next/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Ftcom%2F1512646573344)

**Comment by remisharrock** [comment]
Any testing for this bug ? still here

**Comment by remisharrock** [comment]
the bug persists in the latest version of v5 (as of 14 of february 2018), another test
https://codecast.france-ioi.org/v5/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Ftcom%2F1518650547274

**Comment by remisharrock** [comment]
another long recording that show this:
https://www.google.com/url?q=https://codecast.france-ioi.org/v5/player?base%3Dhttps%253A%252F%252Ffioi-recordings.s3.amazonaws.com%252Fdartmouth%252F1518608621712&sa=D&ust=1518736579498000&usg=AFQjCNEDtoaFtjdVL4zPBOS5o8jCMEQ0xQ

arround 9 minutes and more it's hard to follow because we don't seee which line is being executed

**Comment by remisharrock** [comment]
Codecast recordings impacted:

https://codecast.france-ioi.org/v5/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Fdartmouth%2F1518539100661
around 9 minutes

https://codecast.france-ioi.org/v5/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Fdartmouth%2F1518544519771
around 5 minutes

https://codecast.france-ioi.org/v5/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Fdartmouth%2F1518546503005
around 5min30s

https://codecast.france-ioi.org/v5/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Fdartmouth%2F1518608621712
around 8min

https://codecast.france-ioi.org/v5/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Fdartmouth%2F1518610736869
around 10min30sec

https://codecast.france-ioi.org/v5/player?base=https%3A%2F%2Ffioi-recordings.s3.amazonaws.com%2Fdartmouth%2F1518612236861
around 4min
